src/index.html

```diff
@@ -7,6 +7,49 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--28">
```
Тут за допомогою класів і модифікаторів для контейнера потрібно додати можливість вибирати початковий день для календаря і кількість днів.
Початковий день можна реалізувати через цикл @each (задаючи margin-left для :first-child), а кількість - через цикл @for (задаючи display: none для елементів від 28 до 31)